roles/ansible/tower/config-ansible-tower-license/tasks/license.yml

```diff
@@ -7,15 +7,15 @@
     password: "{{ ansible_tower.admin_password }}"
     force_basic_auth: yes
     method: GET
-    validate_certs: "{{ validate_tower_certs | default(true) }}"
+    validate_certs: "{{ ansible_tower.validate_tower_certs | default(true) }}"
```

Comment:
As this is changing the scope of the variable, there's a risk that old inventories will stop working. Could you please change this to:
```
validate_certs: "{{ ansible_tower.validate_certs | default(validate_tower_certs) | default(true) }}"
```
(**note** the name change of the scoped variable to just `validate_certs`)